submissions/sadoharu/html-css-popup/index.html

```diff
@@ -0,0 +1,70 @@
+    <link rel="apple-touch-icon" sizes="152x152" href="./favicon/apple-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="./favicon/apple-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="192x192"  href="./favicon/android-icon-192x192.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="./favicon/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./favicon/favicon-16x16.png">
+    <link rel="manifest" href="./favicon/manifest.json">
+    <meta name="msapplication-TileColor" content="#ffffff">
+    <meta name="msapplication-TileImage" content="./favicon/ms-icon-144x144.png">
+    <meta name="theme-color" content="#ffffff">
+</head>
+<body>
+<header class="header">
+        <a class="logo" href="#">
+            <img class="logo-img" src="./img/coffee-capuccino-cat-emoji-remixit-cute-food-kawaii-backgrounds-1585244.png" alt="logo">
+            <h1 class="logo-title">Sadoharu training page</h1>
+        </a>
+        <nav class="nav-bar">
+            <ul class="nav-bar_list">
+                <li class="nav-bar_list_item mail"><a class="link" href="#">Mail</a></li>
+                <li class="nav-bar_list_item images"><a class="link" href="#">Images</a></li>
+                <li class="nav-bar_list_item popup-button">
+                    <input class="popup-icon" id="checkbox" type="checkbox">
+                    <label class="icon-grip-solid label" for="checkbox"></label>
+                        <ul class="popup">
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/apps.png" alt="Apps">Apps</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/blogs.png" alt="Blog">Blog</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/calendar.png" alt="Calendar">Calendar</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/chat.png" alt="Chat">Chat</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/classroom.png" alt="Classrrom">Classrrom</a></li>
+                            <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/contacts.png" alt="Contacts">Contacts</a></li>
+                            <li class="popup-showmore">
+                                <input class="popup-more" type="checkbox" id="button">
+                                <label class="popup-lable" for="button"></label>
+                                <ul id="more" class="more" >
+                                    <li class="popup-item"><a class="link" href="#"><img class="img" src="./img/documents.png" alt="Documents">Documents</a></li>
```

Comment:
Read about how to populate `alt` attribute in the right way.
Sorry for russian but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)